runtime/src/iree/hal/drivers/cuda2/cts/CMakeLists.txt

```diff
@@ -8,15 +8,15 @@ iree_hal_cts_test_suite(
   DRIVER_NAME
     cuda2
   DRIVER_REGISTRATION_HDR
-    "experimental/cuda2/registration/driver_module.h"
+    "runtime/src/iree/hal/drivers/cuda2/registration/driver_module.h"
   DRIVER_REGISTRATION_FN
     "iree_hal_cuda2_driver_module_register"
   COMPILER_TARGET_BACKEND
     "cuda"
   EXECUTABLE_FORMAT
     "\"PTXE\""
   DEPS
-    iree::experimental::cuda2::registration
+    iree::hal::drivers::cuda2::registration
   EXCLUDED_TESTS
     # HAL event is unimplemented for now.
     "event"
```

Comment:
Are events still unimplemented?

Reply:
Right. The compiler side does not generate this at the moment; so it's fine.